auctex-latexmk.el

```diff
@@ -54,7 +54,8 @@
 
 ;;; Code:
 
-(require 'tex-buf)
+(unless (string-prefix-p "13" AUCTeX-version)
+  (require 'tex-buf))
```

Comment:
What if AUCTeX is not yet loaded? `AUCTeX-version` will be undefined in that case, right?

How about:

```el
(unless (require 'tex-buf nil t)
  (require 'tex))
```

Reply:
As `latex.el` should be loaded anyway, I think we shall try to load `latex.el` first, then load the `tex-buf.el` according to the version of auctex